Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -411,6 +411,16 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Escaped string for inclusion in a regular expression pattern
+    ///
+    /// "hello ^$ there" -> "hello \\^\\$ there"
+    ///
+    var regexEscaped: String {
+      NSRegularExpression.escapedPattern(for: self)
```

Comment:
I know the return can be omitted but is better add it just to maintain consistency with the others :)

Reply:
I'll toss it in there. BTW, this might be fixed by swiftformat automatically... removing the need for this sort of cleanup :)